src/Compositor.cpp

```diff
@@ -2210,7 +2210,8 @@ void CCompositor::moveWorkspaceToMonitor(PHLWORKSPACE pWorkspace, PHLMONITOR pMo
     // finalize
     if (POLDMON) {
         g_pLayoutManager->getCurrentLayout()->recalculateMonitor(POLDMON->ID);
-        updateFullscreenFadeOnWorkspace(POLDMON->activeWorkspace);
+        if (valid(POLDMON->activeWorkspace))
```

Comment:
valid is redundant for workspaces. a simple bool check should suffice

Reply:
It's used above on activeWorkspace in the same function. Should I get rid of the valid impl for workspaces altogether? Or just switch this usage (and maybe the other usage above?)

Reply:
actually, valid checks inert. This should be fine.